packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Swap/AllowanceCheck/AllowanceCheck.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import {
+  Button,
+  IconCheckCircle,
+  IconQuestionCircle,
+  SemanticColors,
+  Tooltip
+} from '@blockchain-com/constellation'
+import styled, { createGlobalStyle } from 'styled-components'
+
+// TODO: Fix z-index and props and offset in constellation and remove that CSS file
+const GlobalStyleTooltipZIndexFix = createGlobalStyle`
+  div[data-radix-popper-content-wrapper] {
+    z-index: 99999 !important;
+    width: 286px !important;
+    min-width: 286px !important;
+  }
+`
+
+// TODO: Update constellation Button to accept text color
+const GreenButtonWrapper = styled.div`
```

Comment:
Not able to set it directly on Button because it just reset all the styles. So as a temp solution I had to add a wrapper. Will add that prop to constellation later.